src/SDKs/Batch/Support/FileStaging/Batch.FileStaging/Batch.FileStaging.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageId>Azure.Batch.FileStaging</PackageId>
+    <PackageId>Microsoft.Azure.Batch.FileStaging</PackageId>
```

Comment:
@shahabhijeet do we need to publish a new package and publish backlinks to the old package here?